frontend/javascripts/admin/dataset/dataset_upload_view.js

```diff
@@ -349,11 +354,11 @@ class DatasetUploadView extends React.Component<PropsWithFormAndRouter, State> {
     if (files.length === 0) {
       return;
     }
-
     let needsConversion = true;
     for (const file of files) {
       const filenameParts = file.name.split(".");
       const fileExtension = filenameParts[filenameParts.length - 1].toLowerCase();
+      sendAnalyticsEvent("add_files_to_upload", { fileExtension });
```

Comment:
If I drop 100 files, this event will be triggered 100 times. Can we group this by fileExtension? Then, the file count could also be transmitted. E.g.

```js
for (const fileExtension of uniqueFileExtensions) {
  const fileCount = amount of files with that extension
  sendAnalyticsEvent("add_files_to_upload", { fileExtension, fileCount });
}
```

Reply:
Good catch :+1:

I just applied your feedback :)